src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,65 @@
+body {
+    width: 100vw;
```

Comment:
why mentioning specifically its the default behaviour

Reply:
So this **width: 100vw** for my navbar which is coming from CL I gave width 80vw and here I want 100vw